src/main/docker/run-pct.sh

```diff
@@ -39,7 +39,7 @@ if [ -f "${JENKINS_WAR_PATH}" ]; then
   # WAR is accessed many times in the PCT runs, let's keep it local insead of pulling it from a volume
   cp "${JENKINS_WAR_PATH}" "${PCT_TMP}/jenkins.war"
   WAR_PATH_OPT="-war ${PCT_TMP}/jenkins.war "
-  JENKINS_VERSION=$(unzip -q -c "${PCT_TMP}/jenkins.war" META-INF/MANIFEST.MF | grep 'Jenkins-Version' | cut -d ':' -f 2 | tr -d '[:space:]')
+  JENKINS_VERSION=$(groovy /pct/readJenkinsVersion.groovy ${PCT_TMP}/jenkins.war)
```

Comment:
Do not really need Groovy for this.

```bash
unzip -q -c "${PCT_TMP}/jenkins.war" META-INF/MANIFEST.MF | \
  perl -pi -0777 -e 's/\r?\n //g' | \
  grep 'Jenkins-Version' | \
  cut -d ':' -f 2 | \
  tr -d '[:space:]'
```

Reply:
I believe is a matter of tastes, yeah, groovy is overkill, but on the other side, it allows a lot of interesting possibilities like the ability to add custom hooks in the form of groovy scripts at runtime and things like that... Personally, I believe is much easier to understand with the groovy script than the shell, but not really a strong opinion here